examples/keras_io/tensorflow/vision/zero_dce.py

```diff
@@ -0,0 +1,442 @@
+"""
+Title: Zero-DCE for low-light image enhancement
+Author: [Soumik Rakshit](http://github.com/soumik12345)
+Converted to Keras Core by: [Soumik Rakshit](http://github.com/soumik12345)
+Date created: 2021/09/18
+Last modified: 2021/09/19
```

Comment:
Update date

Reply:
Should the `Date created` be modified as well?